site/docs/v1/tech/apis/_apikey-put-request-body.adoc

```diff
@@ -0,0 +1,17 @@
+==== Request Body
```

Comment:
This isn't in `master` but once we merge in the branch we are working on, you can set `keyMaster` on the Update API, but only to downgrade a key. 

So we can add `keyManager` here but the only available option is to set it to `false`, or omit it from the request.